contrib/unit_test_coverage/generate_coverage_report.sh

```diff
@@ -104,7 +107,7 @@ do
     -output-dir=$OUTPUT_DIR/coverage_$COMPONENT \
     -instr-profile=$BUILD_DIR/$COMPONENT.profdata \
     $FORMATTED_BINARY_FILES \
-    $SOURCE_FILES >/dev/null 2>&1
+    $SOURCE_FILES
```

Comment:
Intended?

Reply:
Yes. It produces warnings and I decided to display them. Might help debugging in the future